examples/article-access.amp.html

```diff
@@ -123,6 +123,14 @@
     amp-img {
       background-color: #f4f4f4;
     }
+
+    .login-section {
+      margin: 16px;
+      margin-top: 24px;
+      padding: 16px;
+      background: #ffffcc;
```

Comment:
minor nit (per google css style guide which we don't strictly adhere to anyways), should just be `background: #ffc;`